### ISSUE TITLE
azure:azure-functions-maven-plugin:1.12.0 has incorrect dependencies

### ISSUE THREAD

**Issue by m-moris** [microsoft/azure-maven-plugins#1565]
Currently, I create a Functions application with the following command, the azure functions maven plugin version will be set 1.12.0 in pom.xml.

```
mvn archetype:generate -DarchetypeGroupId=com.microsoft.azure -DarchetypeArtifactId=azure-functions-archetype -DjavaVersion=11
```

And I run `mvn clean package` on the generated project, I get the following error. I set the version to 1.11.0 , it will work correctly.

What is the current correct latest version? 1.12.0 has already been deployed to the Maven Central Repository.

```
$ mvn clean package
[INFO] Scanning for projects...
[INFO]
[INFO] ----------------------< com.example:func-sample >-----------------------
[INFO] Building Azure Java Functions 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.microsoft.azure:azure-functions-maven-plugin:jar:1.12.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.188 s
[INFO] Finished at: 2021-06-03T21:27:33+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin com.microsoft.azure:azure-functions-maven-plugin:1.12.0 or one of its dependencies could not be resolved: Failure to find com.microsoft.azure:azure-functions-maven-plugin:jar:1.12.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

**Comment by Flanker32** [comment]
@m-moris Thanks for your report, we have released the function maven plugin 1.12.0 to maven repository, you may get it [here](https://repo1.maven.org/maven2/com/microsoft/azure/azure-functions-maven-plugin/1.12.0/), could you please try clean the local  repository and re-run maven command with -U?

**Comment by m-moris** [comment]
Thanks for the advice.
I cleaned the local repository and tried again, and it worked correctly, so close this issue.
